app.rb

```diff
@@ -0,0 +1,181 @@
+require 'colorize'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'methods'
+
+class App
+  attr_accessor :rentals, :books, :people
+
+  def initialize
+    @books = []
+    @rentals = []
+    @people = []
+  end
+
+  def options
+    puts 'Enter a corresponding number to chose an option:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+
```

Comment:
- [ ] Please make sure to add a `main.rb` file that will be the entry point of your app and place this list of options in the file and call the methods from the `app.rb` file based on user selection.

![image](https://user-images.githubusercontent.com/83803414/199729437-cecde964-3364-4a7e-bc17-af81bd2c0e07.png)

Reply:
@thecodechaser Thank you so much, I will implement the required changes in the project.